build.py

```diff
@@ -78,7 +78,7 @@
     "ort_openvino_version": "2024.4.0",
     "standalone_openvino_version": "2024.4.0",
     "dcgm_version": "3.3.6",
-    "vllm_version": "0.5.5",
+    "vllm_version": "0.6.3.post1",
```

Comment:
the latest one is 0.6.4.post1. Is the `0.6.3.post1` requested, or we can go higher?

Reply:
0.6.3.post1 is the most tested variant, but we can go higher if needed.

Reply:
sounds good, can keep `0.6.3.post1` for now